gravitee-apim-repository/gravitee-apim-repository-test/src/test/java/io/gravitee/repository/management/EventLatestRepositoryTest.java

```diff
@@ -163,8 +167,9 @@ public void shouldReturnApiEventsInSameOrderWhenSearchWithPagination() {
 
         events = eventLatestRepository.search(eventCriteria, Event.EventProperties.API_ID, 2L, 2L);
         iterator = events.iterator();
-        assertEquals(1L, events.size());
+        assertEquals(2L, events.size());
```

Comment:
Why do you need to change this assertion in regards of your changes ? > This question applies on others tests modifications on this unit test.

Reply:
Because I've added new test cases and the consequence is that these new events I've added are matching for some other tests.
I've double checked and also verify that the exact same dataset work for both jdbc and mongodb.